Composer/packages/lib/code-editor/src/lg/__tests__/TextModalityEditors.test.tsx

```diff
@@ -13,6 +13,7 @@ describe('TextModalityEditor', () => {
   it('should render the value if it is not a template reference', async () => {
     const { findByText } = render(
       <TextModalityEditor
+        focusOnMount={false}
```

Comment:
Why are we setting focusOnMount to false? Should we just remove it?

Reply:
@tdurnford 
linter said it was required, so I set it to false. It's a test.